src/main/java/se/liu/ida/hefquin/engine/federation/access/impl/reqproc/Neo4jRequestProcessorImpl.java

```diff
@@ -0,0 +1,17 @@
+package se.liu.ida.hefquin.engine.federation.access.impl.reqproc;
+
+import se.liu.ida.hefquin.engine.federation.Neo4jServer;
+import se.liu.ida.hefquin.engine.federation.access.*;
+import se.liu.ida.hefquin.engine.federation.access.impl.response.StringResponseImpl;
+
+import java.util.Date;
+
+public class Neo4jRequestProcessorImpl implements Neo4jRequestProcessor{
+    @Override
+    public StringRetrievalResponse performRequest(Neo4jRequest req, Neo4jServer fm) {
+        final Neo4jConnectionFactory.Neo4jConnection conn = Neo4jConnectionFactory.connect(fm.getInterface().getURL());
+        final Date requestStartTime = new Date();
+        String result = conn.executeQuery(req.getCypherQuery());
```

Comment:
```suggestion
    public StringRetrievalResponse performRequest( final Neo4jRequest req, final Neo4jServer fm ) {
        final Date requestStartTime = new Date();
        final Neo4jConnectionFactory.Neo4jConnection conn = Neo4jConnectionFactory.connect( fm.getInterface().getURL() );
        final String result = conn.executeQuery(req.getCypherQuery());
```

Reply:
Another thought here, not for this PR but as a more general thing for the future: We should avoid having to open a new connection for every request. Hence, we need something like a connection manager from which a required connection can be obtained and that takes care of opening and closing the connections on demand.